var/spack/repos/builtin/packages/ior/package.py

```diff
@@ -64,4 +65,9 @@ def configure_args(self):
         else:
             config_args.append("--without-ncmpi")
 
+        if "+ime" in self.spec:
+            config_args.append("--with-ime")
+        else:
+            config_args.append("--without-ime")
+
```

Comment:
You could use `config_args.append(self.with_or_without("time"))` here (see https://spack.readthedocs.io/en/latest/build_systems/autotoolspackage.html#with-or-without).

Reply:
I gave it a try and everything still builds - thanks for pointing this out!